DESCRIPTION

```diff
@@ -61,7 +61,9 @@ Imports:
     knitr (>= 1.12),
     yaml (>= 2.1.5),
     htmltools (>= 0.2.4),
-    caTools
+    caTools,
```

Comment:
FWIW we list 0.9.19 here: https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java#L437

Should we just update the `jsonlite` version throughout?